jupyterhub/utils.py

```diff
@@ -247,6 +247,14 @@ def admin_only(self):
     if user is None or not user.admin:
         raise web.HTTPError(403)
 
+@auth_decorator
+def metrics_authentication(self):
+    """Decorator for restricting access to metrics"""
+    user = self.get_current_user()
+    authenticate_prometheus = self.authenticate_prometheus()
```

Comment:
This should be `self.authenticate_prometheus`, not `self.authenticate_prometheus()` (it's a property, not a method).